src/scripts/main.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+  const divMassege = document.createElement('div');
+  const titleMassege = document.createElement('h2');
+
+  divMassege.className = 'notification';
+  divMassege.classList.add(type);
+
+  body.append(divMassege);
+  body.style.position = 'relative';
+  divMassege.style.position = 'absolute';
+  divMassege.style.top = `${posTop}px`;
+  divMassege.style.right = `${posRight}px`;
```

Comment:
Group commands by it's logic
```suggestion
  body.append(divMassege);
  body.style.position = 'relative';

  divMassege.className = 'notification';
  divMassege.classList.add(type);
  divMassege.style.position = 'absolute';
  divMassege.style.top = `${posTop}px`;
  divMassege.style.right = `${posRight}px`;
```